src/main/java/frc/robot/control/Scheduler.java

```diff
@@ -97,26 +102,38 @@ public void run() {
      * And prevents reuse of "dirtied" Commands
      */
     public void forceEnd() {
+        Logger.append(Logger.LogType.Control, "scheduler", "Scheduler force ended.");
         commandQueue.clear();
+        Logger.append(Logger.LogType.Control, "scheduler", "Scheduler command queue cleared.");
         temp.clear();
+        Logger.append(Logger.LogType.Control, "scheduler", "Scheduler temporary cache cleared.");
```

Comment:
No need to mention the temp cache. It's only there to avoid Concurrent Modification